sample-operator/README.md

```diff
@@ -64,5 +64,5 @@ $ kubectl logs -l app=sample-operator
 ```bash
 kubectl delete -f sample-resource.yaml
 kubectl delete -f sample-operator.yaml
-kubectl delete thirdpartyresource sample.mycompany.io
+kubectl delete crd samples.mycompany.io
 ```
```

Comment:
also, it'd be nice if we can make use of multi-stage docker builds to remove the glide/golang host dependency. but that's a later task :)

Reply:
@jamiehannaford also multi-stage docker is not widely available. when it is we'd likely use it on the rook project too.